src/libs/actions/Report.js

```diff
@@ -142,6 +142,11 @@ function getChatReportName(fullReport, chatType) {
             : '')}`;
     }
 
+    // For a basic policy room, return its original name
+    if (ReportUtils.isPolicyRoom({chatType})) {
+        return fullReport.reportName;
+    }
```

Comment:
The `isDefaultPolicy` does not check for basic policy room, so we need to add this check here too.